src/libstore/tests/derived-path.cc

```diff
@@ -45,12 +65,53 @@ class DerivedPathTest : public LibStoreTest
 {
 };
 
-// FIXME: `RC_GTEST_FIXTURE_PROP` isn't calling `SetUpTestSuite` because it is
-// no a real fixture.
-//
-// See https://github.com/emil-e/rapidcheck/blob/master/doc/gtest.md#rc_gtest_fixture_propfixture-name-args
-TEST_F(DerivedPathTest, force_init)
-{
+TEST_F(DerivedPathTest, opaque) {
```

Comment:
Please add comments stating the intent of each new test. Otherwise contributors will have to do a lot of reverse engineering.